### PR TITLE
Add test cluster check when starting csi proxy

### PR DESCRIPTION
**Pull request by jingxu97**
remove the check for test cluster when downloading csi proxy

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 1]
```diff
@@ -930,14 +928,13 @@ function Configure-GcePdTools {
...
-    if (ShouldWrite-File ${env:K8S_DIR}\diskutil.exe) {
-      # The source code of this executable file is https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/cmd/diskutil/diskutil.c
-      MustDownload-File -OutFile ${env:K8S_DIR}\diskutil.exe `
-        -URLs "https://ddebroywin1.s3-us-west-2.amazonaws.com/diskutil.exe"
-    }
-    Copy-Item ${env:K8S_DIR}\diskutil.exe -Destination "C:\Windows\system32"
+  if (ShouldWrite-File ${env:K8S_DIR}\diskutil.exe) {
+    # The source code of this executable file is https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/cmd/diskutil/diskutil.c
+    MustDownload-File -OutFile ${env:K8S_DIR}\diskutil.exe `
+      -URLs "https://ddebroywin1.s3-us-west-2.amazonaws.com/diskutil.exe"
```
Comment (yliaog) [inline cluster/gce/windows/k8s-node-setup.psm1:934]: it's kind of ok to download from the above URL if the cluster is for test, it is not ok if it is not for test.

Comment (jingxu97) [reply cluster/gce/windows/k8s-node-setup.psm1]: we recently added an API in csi-proxy which will remove the need of this diskutil. I will update driver and remove this part after testing.

**Comment by jingxu97** [comment]
@yliaog @pjh I updated the PR. PTAL

**Comment by pjh** [comment]
Thanks Jing. So this is now just adding a check that we missed the first time around?

Was there anything else you wanted this PR to do? If we can remove the dependency on diskutil.exe like you said then that would be good, but if we can't do it now then we can just submit this.

**Comment by jingxu97** [comment]
> Thanks Jing. So this is now just adding a check that we missed the first time around?
> 
> Was there anything else you wanted this PR to do? If we can remove the dependency on diskutil.exe like you said then that would be good, but if we can't do it now then we can just submit this.

Now csi-proxy can handle getDiskName already. But I am waiting for a new release cut so that our build and release pipeline can get the latest version. It should be done this week.

**Comment by jingxu97** [comment]
@pjh @yliaog PTAL. Thanks!

**Comment by pjh** [comment]
/lgtm

Thanks!

**Pull request merged**